programs/tokenized-vaults-program/src/state/investor_escrow.rs

```diff
@@ -0,0 +1,43 @@
+use anchor_lang::prelude::*;
+
+pub const INVESTOR_ESCROW_SEED: &str = "investor_escrow:";
+
+#[derive(Default, Debug, InitSpace)]
+#[account(discriminator = 3)]
+pub struct InvestorEscrow {
+    pub authority: Pubkey,
+    pub mint: Pubkey,
```

Comment:
As long as , we know the USDC mint from constants, do you think it's necessary to save it?